doc/source/security/enabling-master-key.rst

```diff
@@ -0,0 +1,214 @@
+.. _enabling-master-key:
+
+==============================================================================
+Enabling Master Key Encryption
+==============================================================================
+
+.. _enabling-vault:
+
+Enabling the Keyring Vault
+---------------------------
+
+To enable Master key vault encryption, the user must have
+`SYSTEM_VARIABLES_ADMIN
+<https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_system-variables-admin>`__
+privileges.
+
+Add the following statements to my.cnf:
+
+.. code-block:: MySQL
+
+    [mysqld]
+    early-plugin-load="keyring_valut=keyring_valut.so"
+    loose-keyring_value_config="/home/mysql/keyring_vault.conf"
+
+Restart MySQL.
+
+To change the default value, use the following syntax:
+
+.. code-block:: guess
+
+   mysql> SET innodb_default_encryption_key_id = NEW_ID
```

Comment:
innodb_default_encryption_key_id is part of KEYRING encryption, should not be here.

Reply:
removed.